kong/plugins/aws-lambda/handler.lua

```diff
@@ -260,20 +230,11 @@ function AWSLambdaHandler:access(conf)
     end
   end
 
-  local ctx = ngx.ctx
-  if ctx.delay_response and not ctx.delayed_response then
-    ctx.delayed_response = {
-      status_code = status,
-      content     = content,
-      headers     = headers,
-    }
-
-    ctx.delayed_response_callback = flush
-
-    return
+  if kong.configuration.enabled_headers["Via"] then
```

Comment:
We should keep using the value from `constants.HEADERS.VIA` here. While I am not a fan of those, in this case it is the pattern already used across the codebase (consistency) and it eliminates concerns about the case-sensitivity which initially had me raise my eyebrows here.